src/docs/development/add-to-app/android/add-flutter-fragment.md

```diff
@@ -255,6 +291,20 @@ Please see the [instructions for instantiating and starting a `FlutterEngine`].
 To use a pre-warmed `FlutterEngine` in a `FlutterFragment`, instantiate a
```

Comment:
line 289 doesn't work. I don't think such link exists.

Reply:
It looks like this branch has 
docs/development/add-to-app/android/add-splash-screen
which may be relevant.

Reply:
I removed the link for now.